requirements.txt

```diff
@@ -1,11 +1,11 @@
 Flask==1.1.2
-python-dotenv==0.14.0
+python-dotenv==0.19.2
```

Comment:
If possible, I'd like to set these as continously updated.  So

`python-dotenv>=0.19.2`

Reply:
@jberkus I've the set the packages to continuous updates and have also upgraded the rest of them. Please have a look at the latest commit.